simulariumio/tests/test_agents_from_lists.py

```diff
@@ -0,0 +1,493 @@
+import numpy as np
+
+from simulariumio import AgentData
+
+
+def test_agent_data_jagged_list():
+    list_data = {
+        "times": [0.0, 0.1, 0.2],
+        "n_agents": [3, 4, 3],
+        "viz_types": [
+            [1000.0, 1001.0, 1001.0],
+            [1001.0, 1000.0, 1001.0, 1001.0],
+            [1000.0, 1000.0, 1001.0],
+        ],
+        "unique_ids": [
+            [0.0, 1.0, 2.0],
+            [0.0, 1.0, 2.0, 3.0],
+            [0.0, 1.0, 2.0],
+        ],
+        "type_names": [
+            ["H", "A", "C"],
+            ["L", "D", "A", "U"],
+            ["E", "Q", "K"],
+        ],
+        "positions": [
+            [
+                [4.89610492, -29.81564851, 40.77254057],
```

Comment:
can some of this data be defined as variables so you don't have to repeat it inline here?